### PR TITLE
NLLB evaluation script

### PR DESCRIPTION
**Pull request by Tundeh**
This pull request implements the evaluation script for the NLLB.

### CODE REVIEW

**Comment by heisguyy** [comment]
Hi Tundeh can you provide an example of how to use the script.

File: src/finetining/NLLB/eval.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+    model, tokenizer = load_model(model_name)
+    data = process_file(dataset_path=dataset_path)
+    sorted_data = sorted(data, key= lambda item: len(item["translation"][source_lang]), reverse=True)
+
+    result = []
+    for i in trange(0, len(sorted_data), batch_size):
+
+        sorted_batch = sorted_data[i: i + batch_size]
+        batch = [item["translation"][source_lang] for item in sorted_batch]
```
Comment (owos) [inline src/finetining/NLLB/eval.py:66]: this looks good. we might be using the tsv format on gitbub too. I will drop the finetuning script for you to see soon.

**Review by owos — APPROVED** [review]

**Comment by Tundeh** [comment]
> Hi Tundeh can you provide an example of how to use the script.

Run the script and pass the arguments in the command line like this `python3 eval.py --model_name facebook/nllb-200-distilled-600M --dataset_path ../data/yoruba_translation --source_lang en --target_lang yor`

File: src/finetining/NLLB/eval.py (hunk 1 above)
Comment (Tundeh) [reply src/finetining/NLLB/eval.py:66]: Okay

**Pull request merged**